Needs a C compiler - not a C++ compiler

$ diff -urN threaded-port-scanner-1.3/Makefile.orig threaded-port-scanner-1.3/Makefile
--- threaded-port-scanner-1.3/Makefile.orig	2022-03-10 18:26:08.589870300 +0100
+++ threaded-port-scanner-1.3/Makefile	2022-03-10 18:27:22.299557300 +0100
@@ -1,17 +1,17 @@
-CXX := gcc
-CXXFLAGS := -std=gnu99 -O2 -Wall -ggdb
+CC ?= gcc
+CFLAGS := -std=gnu99 -O2 -Wall -ggdb
     
 OBJECTS	:= main.o net.o pathtools.o
     
 pscan: $(OBJECTS)
-	$(CXX) $(OBJECTS) -lws2_32 -o pscan
+	$(CC) $(OBJECTS) -lws2_32 -o pscan
     
 main.o: main.c pathtools.h
-	$(CXX) $(INCLUDES) $(CXXFLAGS) -c main.c -o main.o
+	$(CC) $(INCLUDES) $(CFLAGS) -c main.c -o main.o
     
 net.o: net.c
-	$(CXX) $(CXXFLAGS) -c net.c -o net.o
+	$(CC) $(CFLAGS) -c net.c -o net.o
 
 pathtools.o: pathtools.c pathtools.h
-	$(CXX) $(CXXFLAGS) -c pathtools.c -o pathtools.o
+	$(CC) $(CFLAGS) -c pathtools.c -o pathtools.o
 
